qa/tasks/cephfs/test_mirroring.py

```diff
@@ -194,6 +194,16 @@ def remove_directory(self, fs_name, fs_id, dir_name):
 
         self.assertLess(vafter["counters"]["directory_count"], vbefore["counters"]["directory_count"])
 
+    def list_directories(self, fs_name, fs_id):
+        self.run_ceph_cmd("fs", "snapshot", "mirror", "ls", fs_name)
+        dirs_list = json.loads(self.get_ceph_cmd_stdout("fs", "snapshot", "mirror", "ls", fs_name))
+
+        # verify via asok
+        res = self.mirror_daemon_command(f'mirror status for fs: {fs_name}',
+                                         'fs', 'mirror', 'status', f'{fs_name}@{fs_id}')
+        dir_count = res['snap_dirs']['dir_count']
+        self.assertTrue(len(dirs_list) == dir_count)
```

Comment:
This should additionally compare if the directories on which mirroring is set matches the one returned by the command.

Reply:
Then we need to change the command to `peer status`.

Reply:
I don't get it. All that's needs to be done is to record which directory paths mirroring is being added and then check if those paths show up in the output.

Reply:
updated